src/components/transactions/transactionType.js

```diff
@@ -13,7 +13,7 @@ const TransactionType = (props) => {
       type = t('Second Passphrase Registration');
```

Comment:
It wasn't on the list, but to be consistent all other transaction types should also have only the first letter capital, e.g.: `Second Passphrase Registration` -> `Second passphrase registration`